src/devices/ServoMotor/README.md

```diff
@@ -0,0 +1,69 @@
+﻿# Servo Motor
+
+## Summary
+This is a generic binding to control many servo motors using a hardware or software `PwmChannel`.  Servo motors are usually based on a signal frequency of 50Hz.  They also require a minimum/maximum pulse width to determine the position.  The pulse width is generally between 1 and 2 milliseconds, where 1ms is approximately 0 degrees, 1.5ms is the rest position, and 2ms is 180 degrees.  This information can be found in each servo motor's datasheet.  
+
+One thing to be aware of is the wiring as the servo motor connector is usually a 3-pin connector.  The pinout is shown below where colors can vary.
+
+| Pin Number | Signal         | Color                     |
+|------------|----------------|---------------------------|
+| 1          | Ground         | Black or Brown            |
+| 2          | Power Supply   | Brown or Red              |
+| 3          | Control Signal | Orange or White or Yellow |
+
+
+## Device Family
+There are many servo motor sizes available that offer both standard and continuous rotation.  Below are a few links where to purchase servo motors.
+
+[Adafruit Servo Motor Accessories](https://www.adafruit.com/?q=servo)  
+[Sparkfun Servo Motor Accessories](https://www.sparkfun.com/categories/245)  
+
+## Binding Notes
+
+The `ServoMotor` binding offers an easy way to begin controlling a servo motor.  The quickest approach is to provide the `ServoMotor` object a `PwmChannel` using the default values for other optional arguments.
+
+```csharp
+// Example of hardware PWM using chip 0 and channel 0 on a dev board.
+ServoMotor servoMotor = new ServoMotor(PwmChannel.Create(0, 0, 50));
+servoMotor.Start();  // Enable control signal.
```

Comment:
what is this actually starting? will servo motor do anything after this call?

Reply:
Start() method just starts the pulse width on channel that sends signal to servo motor.  The motor will adjust to position based on current frequency and duty cycle.  Once started, the properties can be changed and position will adjust without having to call Start() again.

The signal could automatically start (when instantiating).  However, it is similar to the PwmChannel, where a user might want to create the object first without sending pulses and then start/stop to do things without disposing.  This would be critical in some scenarios and bad things could happen if it was coded using some of the default frequencies & duty cycles.  For example... a model airplane with servo throttle 🚀

Reply:
I was actually trying to get rid of Start/Stop on PWM because PWM with 0% fill is the same as stopped PWM but seems we might be too far in the release cycle for that at the moment.

Presumably when you don't start PWM then PWM fill will be 0 or 1 (or basically all high or low).

Would that have any effect on the servo? I.e. if servo was set at `MinAngle + 30% * AngleRange` before it was powerd up and I powered it up to line with control signal with 0% fill would that have any effect on servo? (i.e. servo going to MinAngle)

If Start/Stop have observable difference for user I'm fine with having them but if no movement is equivalent to some setting I'd rather not have it.

Reply:
> Presumably when you don't start PWM then PWM fill will be 0 or 1 (or basically all high or low).

This would be a bad move for a few reasons.  Below are some I could think of...
- Some PWM devices don't allow for 100% duty cycle.  For example, I've seen some potential binding devices that throw errors when writing 100% DC to registers.  I'm not sure, but might be case for 0%, as well.  Those same bindings also have flags in registers to enable/disable the channels' signals.
- Users would have to remember to write 0/1 to disable and then remember the value they were using beforehand (e.g. 0.452, etc.).  Stop/Start appears better and you don't have to remember as the device memory does it for you automatically.
- Start/Stop seems to be used in other IoT APIs including WinRT.  Maybe we could use Enable/Disable.  I don't want to get hung on a particular name (but be similar across PWM APIs; core, motors, buzzers, LEDs,, etc.).  I'm just more set on having the API ability to create the PWM related objects without starting signal immediately and then being able to enable/disable the signal when ready.